rockcraft/models/project.py

```diff
@@ -179,15 +187,14 @@ class NameStr(pydantic.ConstrainedStr):
     regex = re.compile(NAME_REGEX)
 
 
-class Project(YamlModel):
+class Project(YamlModelMixin, BaseProject):
```

Comment:
`YamlModelMixin` is to get the `yaml()` method; possibly we should put this in the superclass

Reply:
Agreed, superclass seems to be the way to go.